src/libratbag-data.c

```diff
@@ -160,6 +163,9 @@ init_data_hidpp20(struct ratbag *ratbag,
 		data->hidpp20.index = num;
 	if (error)
 		g_error_free(error);
+
+	str = g_key_file_get_string(keyfile, group, "Quirk", NULL);
```

Comment:
move that to the next commit please, the compiler should complain about this being unused.